extensions/wikia/VideoHandlers/tests/VideoFileUploaderTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * Class VideoFileUploaderTest
+ *
+ * @group MediaFeatures
+ */
+class VideoFileUploaderTest extends WikiaBaseTest {
+
+	/**
+	 * @dataProvider destinationTitleDataProvider
+	 */
+	public function testGetNormalizedDestinationTitle( $areIdentical, $title, $expectedTitle ) {
+		if ( $areIdentical ) {
```

Comment:
So this is essentially a test of PHPs `==` operator, right?  This probably isn't necessary since the issue was never with PHPs internal handling of UTF8 bytes.